components/ProfilePage/Profile.tsx

```diff
@@ -200,15 +203,17 @@ export const ProfileAboutSection = ({
   const { twitter, linkedIn }: { twitter?: string; linkedIn?: string } =
     profile?.social ?? {}
 
+  const { t } = useTranslation("profile")
+
   return (
     <MapleCard
-      header={`About ${profile?.displayName?.split(" ")[0] ?? "User"}`}
+      header={t("aboutHeader", { firstName: profile?.displayName?.split(" ")[0] ?? "User" }).toString() ?? ""}
```

Comment:
Maybe out of an abundance of caution to avoid runtime errors you could also use the [`?.[0]` optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining#optional_chaining_with_expressions)? Super nit though. Also not sure if the `toString()` is necessary as it looks like `t()` returns a string itself.
```suggestion
      header={t("aboutHeader", { firstName: profile?.displayName?.split(" ")?.[0] ?? "User" }) ?? ""}
```

Reply:
Hi Tom, you're right. We don't need the `toString`